package.json

```diff
@@ -39,7 +41,7 @@
     "ts-jest": "^28.0.0"
   },
   "engines": {
-    "node": ">=16.0.0 <17.0.0"
+    "node": ">=16.0.0 <=18.16.1"
```

Comment:
Don't we want to exclude node 17, which is end of life ?

Reply:
```suggestion
    "node": ">=16.0.0 <17.0.0"
```